### PR TITLE
air720 update And bc26 sleep mode fix

### PR DESCRIPTION
**Pull request by shao7936626**
->air720
1.cat.1 compatible;
2.fix bugs
3.update at commands relay to the air720_at_manual 4.20

->bc26
1.fix bug (power key retain low , when the module comes into sleep mode,so the next time,when module wakes up ,the module can't come into sleep mode )

### CODE REVIEW

File: class/air720/at_device_air720.c [hunk 1]
```diff
@@ -501,18 +491,18 @@ static int air720_netdev_ping(struct netdev *netdev, const char *host,
         return -RT_ERROR;
     }
 
-    for (i = 0; i < rt_strlen(host) && !isalpha(host[i]); i++)
-        ;
+    // for (i = 0; i < rt_strlen(host) && !isalpha(host[i]); i++)
```
Comment (Lawlieta) [inline class/air720/at_device_air720.c:494]: 确定无用的代码可以直接删除

File: class/air720/at_socket_air720.c [hunk 2]
```diff
@@ -339,6 +337,10 @@ static int air720_socket_send(struct at_socket *socket, const char *buff, size_t
     return result;
 }
 
+static int air720_socket_quick_send(struct at_socket *socket, const char *buff, size_t bfsz, enum at_socket_type type)
```
Comment (Lawlieta) [inline class/air720/at_socket_air720.c:340]: 这个函数的作用？TODO 或者删除

File: class/bc26/at_device_bc26.c [hunk 3]
```diff
@@ -165,6 +165,7 @@ static int bc26_wakeup(struct at_device *device)
         rt_thread_mdelay(100);
         rt_pin_write(bc26->power_pin, PIN_LOW);
         rt_thread_mdelay(200);
+		rt_pin_write(bc26->power_pin, PIN_HIGH);
```
Comment (Lawlieta) [inline class/bc26/at_device_bc26.c:168]: 注意对齐

Comment (shao7936626) [reply class/bc26/at_device_bc26.c]: 更新了

**Pull request merged**